word_bank.rb

```diff
@@ -0,0 +1,124 @@
+def initialize_word_bank
+  return ["ear",
+  "heart",
+  "irritate",
```

Comment:
I love that you've separated your words out into a separate file. This makes it very clear which part of the program is which, and the reader doesn't have to scroll through a hundred lines of words at the top of your file.